psycopg/setup.cfg

```
@@ -43,5 +44,32 @@ install_requires =
     typing-extensions >= 4.1
     tzdata; sys_platform == "win32"
```

As you can see from [the ci failure](https://github.com/psycopg/psycopg/actions/runs/3750941955/jobs/6371310179), importlib.metadata is not available on Python 3.7. We should add an entry in the `install_requires` section and install the backport module [`importlib_metadata`](https://pypi.org/project/importlib-metadata/), such as:

```suggestion
    tzdata; sys_platform == "win32"
    importlib-metadata >= 1.4; python_version < "3.8"
```

(I propose >= 1.4 because that ensures compatibility with Python 3.8, according to the versions compatibility they have in their readme).